website/content/en/docs/getting-started/_index.md

```diff
@@ -236,6 +223,10 @@ spec:
     resources:
       cpu: 1000
   provider:
+    subnetSelector:
```

Comment:
Now that we don't autodiscover, we should be telling customers how to choose a subnetSelector and a securityGroupSelector in our documentation. Maybe a line or two. 
 
We could use EKS as an example in our docs since this is AWS specific so should make sense to most customers. We could reference the clusterWorkerSG and cluster's subnets as good examples to select here 

Also do we [need to edit this para](https://github.com/aws/karpenter/blob/main/website/content/en/docs/getting-started/_index.md?plain=1#L212)?

Reply:
done